packages/teams-js/src/public/meeting.ts

```diff
@@ -210,6 +210,30 @@ export namespace meeting {
     error?: SdkError;
   }
 
+  /**
+   * Interface for mic state change
+   *
+   * @beta
+   */
+  export interface MicState {
```

Comment:
As I mentioned below, it would be better to have an interface that only returns the correct values to the app developer and throws SdkErrors if something exceptional is received from the host.